.github/workflows/teleport-create-role.yaml

```diff
@@ -0,0 +1,54 @@
+name: Create teleport role
+on:
+  workflow_call:
+      inputs:
+        cluster_teleport:
+          description: "Address teleport cluster"
```

Comment:
typo :D it's either "address of the teleport cluster" or "teleport cluster address"